logbook-spring-boot-autoconfigure/pom.xml

```diff
@@ -104,6 +104,16 @@
             <version>${spring-boot.version}</version>
             <scope>provided</scope>
         </dependency>
+        <!--
+            As of Spring Framework 6.1, SnakeYAML 2.0 or higher is required.
+            It's set to optional since Spring Boot 2.x requires SnakeYAML 1.x.
+        -->
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>2.1</version>
```

Comment:
Why do we need to explicitly specify the dependency? I'm surprised that spring-boot-starter [uses 1.33](https://github.com/spring-projects/spring-boot/blob/v3.1.5/spring-boot-project/spring-boot-dependencies/build.gradle#L1376-L1380) version still... Looks like versions [3.2.x will have the newer version](https://github.com/spring-projects/spring-boot/blob/v3.2.0-M1/spring-boot-project/spring-boot-dependencies/build.gradle#L1365-L1370).